SpatialGDK/Source/SpatialGDKServices/Public/LocalDeploymentManager.h

```diff
@@ -19,6 +19,7 @@ class FLocalDeploymentManager
 {
 public:
 	FLocalDeploymentManager();
+	~FLocalDeploymentManager();
```

Comment:
class is nether final nor destructor virtual?

Reply:
Oooooooh, nice point! I don't think we have concerned ourselves with final in the past. A nice habit to get into though!

Reply:
that makes sense as a practice to get into, I've made the class final 😄